### DISCUSSION TITLE
Short breaks looks weird in lazer

### DISCUSSION THREAD

**Discussion by tutis12** [ppy/osu#30888] [Ideas]
This map: https://osu.ppy.sh/beatmapsets/2228966#osu/4746348

The break overlay is almost flashing, I don't think this is intended.

https://github.com/user-attachments/assets/63ca9b6d-ee85-4837-bc69-a838447fb3ae

**Comment by peppy** [comment]
Something to definitely be considered when we get to redesigning the break overlay.

From recollection, stable will just not display a break if it's short enough (or maybe it does in a case like this but there's no aggressive movement animation which makes it look jarring).

**Comment by fleuph** [comment]
For the example, here is what that particular map looks like in stable

With "don't change dim level during breaks" DISABLED:
https://github.com/user-attachments/assets/02dc43fe-0610-49b2-be01-134584094a97

With "don't change dim level during breaks" ENABLED
https://github.com/user-attachments/assets/86df45de-5269-4fa1-b9a9-c530a118af9b
